models/members.js

```diff
@@ -13,21 +13,26 @@ const userModel = firestore.collection('users')
 
 const fetchMembers = async () => {
   try {
-    const snapshot = await userModel.where('isMember', '==', true).get()
+    const snapshot = await userModel.get()
 
-    const allMembers = []
+    const allMembers = { oldMembers: [], newMembers: [] }
 
     if (!snapshot.empty) {
       snapshot.forEach((doc) => {
-        allMembers.push({
+        const memberData = doc.data()
+        const curatedMemberData = {
           id: doc.id,
-          ...doc.data(),
+          ...memberData,
           tokens: undefined,
           phone: undefined,
           email: undefined
-        })
-      }
-      )
+        }
+        if (typeof memberData.roles === 'undefined') {
+          allMembers.newMembers.push(curatedMemberData)
+        } else if (memberData.roles && memberData.roles.member) {
+          allMembers.oldMembers.push(curatedMemberData)
+        }
```

Comment:
I think we can re-structure this to something like this

```javascript
if (memberData.roles && memberData.roles.member) {
allmembers.oldMembers.push(curatedMemberData)
} else {
...new members
}
```

Reply:
Done with the update.